README.md

```diff
@@ -1,50 +1,71 @@
-A third option is to run the command pallete (CMD+SHIFT+P on Mac or equivalent) and look for the `MythX: Analyze smart contract` command.
+If you do want to use a ethereum address/password combination the extension supports it. Please enter your registered MythX Ethereum address and password in the VS Code user settings. These fields are properties of the extension as shown in the screenshots below:
 
-![button_one](images/screenshots/button_mythx.png)
-![right_click](images/screenshots/right_click.png)
+After installing the extension please enter your registered MythX `ethAddress` and `password` in VSCode user settings. These fields are properties of `mythxvsc` as shown in the screenshots below.
 
-Once solidity compilation is done you will be prompted to enter the name of the contract you want to analyze from a list of contracts found in the solidity file. It is very important that you pick the main contract name or the API will return an error and your analysis will fail.
+If no `ethAddress` and `password` provided the extension will fallback to use default trial credentials for MythX (trials account will soon be deprecated).
+
+![ethaddress_login](images/screenshots/ethaddress_login.png)
+
+**Please note that the credentials stored this way are exposed to VS Code. Be sure to understand the security risk this entails or contact the extension developers if you don’t.**
+
+# Usage
+
+MythX give their users the ability to choose from three different scan options. This extension supports all three:
+
+-   Quick analysis. This analysis will take up to 2 minutes. **This is the default functionality and also gives you linting formatting results in vscode. Just click on the MythX icon on the top right of the IDE window.**
+
+-   Standard analysis. This will take up to 30 minutes.
+
+-   Deep analyisis. This will take up to 90 minutes.
+
+![analysis_mode](images/screenshots/analysis_mode.png)
+
+Once the solidity compilation is done, if there is only one contract in the solidity file you can just wait for analysis results. If there are multiple contracts in the same file you will be asked to pick a contract from a dropdown list of contracts that exist in the compiled AST. Please make sure to pick the main contract to avoid inconsistent results.
 
 ![contract_name](images/screenshots/contract_picker.png)
 
-Now you can just seat back and wait for MythX to do its magic :) Once analysis is over you will see your smart contract issues highlighted in your code. This should take no longer than three minutes.
+You will also get a popup window with a link to the dashboard that will take straight to your results, or if using quick mode you will see highlighted issues in your solidity code. The exact time it takes to perform an analysis will depend on the plan associated with your account.
```

Comment:
"You will also get a popup window with a link to the dashboard that will take you straight to your results, or if using quick mode you will see highlighted issues in your solidity code."